cloudinit/util.py

```diff
@@ -490,6 +490,12 @@ def multi_log(
 
 @lru_cache()
 def is_Linux():
+    """deprecated: prefer Distro object's `is_linux` property
```

Comment:
We could just drop this function altogether if we can figure a better approach to sort `cloud-init  analyze dump` as the only other use-case right?

Reply:
yep!